client-side/src/App.jsx

```diff
@@ -1,18 +1,16 @@
-import { Provider } from 'react-redux';
-import { RouterProvider } from 'react-router-dom';
-import Footer from './stories/footer/FooterComponent';
-import { router } from './router/router.jsx';
-import { store } from './redux/store.jsx';
+import React from 'react';
+import Header from './stories/header/header'
+import Footer from './stories/footer/FooterComponent'
+import DateTimePicker from './components/report';
 import './App.scss';
 
 function App() {
   return (
-    <>
```

Comment:
remove your changes from this file, you can copy its content from main branch in git